internal/pkg/service/cli/cmd/local/cmd.go

```diff
@@ -16,11 +21,11 @@ func Commands(p dependencies.Provider, envs *env.Map) *cobra.Command {
 		Long:  helpmsg.Read(`local/long`),
 	}
 	cmd.AddCommand(
-		CreateCommand(p),
-		PersistCommand(p),
-		EncryptCommand(p),
-		ValidateCommand(p),
-		FixPathsCommand(p),
+		create.CreateCommand(p),
+		persist.Command(p),
+		encrypt.Command(p),
+		validate.ValidateCommand(p),
```

Comment:
In each package is one command now, so please fix naming, `validate.ValidateCommand` -> `validate.Command`, `workflow.WorkflowCommand` -> `workflow.Command`, etc...